ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/results-export/results-export.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import { hot } from 'react-hot-loader'
+import { Button, buttonTypeEnum } from '../../presentation/button'
+import styled from '../../styles/styled-components'
+const { Menu, MenuItem } = require('../../menu')
+const Dropdown = require('../../dropdown')
+
+const Root = styled<{}, 'div'>('div')`
+  padding: 50px;
+
+  button {
+    margin-top: ${props => props.theme.minimumSpacing};
+    width: 100%;
+  }
+
+  .export-option {
+    margin-bottom: 50px;
+  }
+`
+
+type ExportFormat = {
+  id: string
+  displayName: string
+}
+
+type Props = {
+  selectedFormat: string
+  exportFormats: ExportFormat[]
+  downloadDisabled: boolean
+  onDownloadClick: () => void
+  handleExportOptionChange: () => void
+}
+
+type State = {}
+
+class ResultsExport extends React.Component<Props, State> {
```

Comment:
This can be simplified to a functional component as State is not used
https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/map-settings/example-coordinates.tsx